### PR TITLE
show ws tabs when editing tool

### LINKED ISSUE #1097: Browser tab title for a jupyter notebook is the workspace name
I have a few notebooks open from the same workspace and they all just use the workspace name as the tab title:

screenshot 2019-01-21 at 8 15 23 pm

This makes finding a particular notebook difficult. In NHS and AMP-PD workspaces, Nicole and Matt created starter/setup notebooks that should be run first. I imagine many users following suit by compartmentalizing code into separate notebooks that they may be working on simultaneously.

### LINKED ISSUE #1368: top nav disappears on tool screen
When I select a tool in the Terra UI the top nav bar disappears and it is hard to figure out how to go back. I now see that there is an X to click on the top right but that is not intuitive. I would expect an X to close out a dialog and this page does not feel like a dialog at all. It feels like a complete page load and it covers the entire screen.

### PR DESCRIPTION
**Pull request by zarsky-broad**
closes #1368 
closes #1097
partially addresses: #1488, #1197, #1387

### CODE REVIEW

**Comment by panentheos** [comment]
Just checked with Brad, he doesn't think we should move forward with this, since it would take substantial real-estate away from notebooks (in particular).

**Comment by zarsky-broad** [comment]
I'll chat with him tomorrow, I think it's a good idea but I can certainly be swayed.

**Comment by zarsky-broad** [comment]
This is ready for a second look, after having gone through it with Brad.

File: src/pages/workspaces/workspace/tools/WorkflowView.js [hunk 1]
```diff
@@ -679,13 +678,6 @@ const WorkflowView = _.flow(
               `Fill in the attributes below to add or update columns in your data table`
             ])
           ])
-        ]),
-        div({ style: { flex: 'none', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/tools/WorkflowView.js:683]: Brad says this should go.

File: src/pages/workspaces/workspace/tools/WorkflowView.js [hunk 2]
```diff
@@ -565,14 +564,14 @@ const WorkflowView = _.flow(
...
-              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 60 } }, [
+              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 75 } }, [
                 h(Select, {
                   isDisabled: !!Utils.editWorkspaceError(ws),
                   isClearable: false,
                   isSearchable: false,
                   value: methodVersion,
                   getOptionLabel: ({ value }) => Utils.normalizeLabel(value),
-                  options: _.uniq([...snapshotIds, savedConfig.methodRepoMethod.methodVersion]).sort(),
+                  options: _.sortBy(_.toNumber, _.uniq([...snapshotIds, savedConfig.methodRepoMethod.methodVersion])),
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/tools/WorkflowView.js:574]: So the list goes `1, 2, 11` instead of `1, 11, 2`

Comment (panentheos) [reply src/pages/workspaces/workspace/tools/WorkflowView.js:574]: Are these always numbers?

**Review by panentheos — APPROVED** [review]
Looks good.

**Review by elizzhao — APPROVED** [review]
Looks good to me!

File: src/pages/workspaces/workspace/tools/WorkflowView.js (hunk 2 above)
Comment (zarsky-broad) [reply src/pages/workspaces/workspace/tools/WorkflowView.js:574]: Apparently in agora they're always int's.

**Pull request merged**